src/graphics/tmp_structs.h

```diff
@@ -0,0 +1,30 @@
+#ifndef TMP_STRUCTS_H
+#define TMP_STRUCTS_H
+
+#include "../../include/raylib.h"
+typedef struct MenuButton{
```

Comment:
We can declare struct without the name after the keyword `struct`. So, a struct declaration with typedef would look like:
```c
typedef struct {
    // data in struct
} MenuButton
```